types/src/on_chain_config/registered_currencies.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{on_chain_config::OnChainConfig, validator_info::ValidatorInfo};
+use move_core_types::identifier::Identifier;
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
+pub struct RegisteredCurrencies {
```

Comment:
it doesn't look like this module is imported/visible in on_chain_config/mod.rs. So not accessible from other crates

Reply:
Good spot! I'll update now :)